installer/lib/phoenix_new.ex

```diff
@@ -153,7 +164,8 @@ defmodule Mix.Tasks.Phoenix.New do
   def run(app, mod, path, opts) do
     db = Keyword.get(opts, :database, "postgres")
     ecto = Keyword.get(opts, :ecto, true)
-    brunch = Keyword.get(opts, :brunch, true)
+    html = Keyword.get(opts, :html, true)
+    brunch = Keyword.get(opts, :brunch, html && true)
```

Comment:
I am not sure if we should do this. I can have my javascript app all written with brunch and just use json. Maybe we do need an `--api` flag that is a shortcut for `--no-html` and `--no-brunch`?